docs/prerequisites.rst

```diff
@@ -0,0 +1,77 @@
+Prerequisites
+===============
+
+- Tezos
+
+Mac:
+Brew:
+
+```
+ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+````
+
+External libs:
+```
+brew install hidapi libev wget
+```
+
+Install tezos:
+```
+git clone https://gitlab.com/tezos/tezos.git
+cd tezos
+git checkout latest-release
+```
+
+Install Ocaml:
+```
+brew install ocaml
+wget https://github.com/ocaml/opam/releases/download/2.0.7/opam-2.0.7-x86_64-macOS
+sudo mv opam-2.0.7-x86_64-macOS /usr/local/bin/opam
+cd /usr/local/bin/
+sudo chmod a+x opam
+opam init
+opam update
+opam update
+eval $(opam env)
+opam switch create 4.09.1
```

Comment:
wget -O latest-release:version.sh https://gitlab.com/tezos/tezos/raw/latest-release/scripts/version.sh
source latest-release:version.sh
opam switch create for_tezos $ocaml_version
eval $(opam env)

This would be an instruction not getting outdated